### PR TITLE
Set aria label on subnavigation

### PR DESCRIPTION
**Pull request by Katrienvh**
## Type
- Enhancement

## Pull request description
Set aria label on subnavigation based on the active page

### CODE REVIEW

File: src/Backend/Core/Layout/Templates/base.html.twig [hunk 1]
```diff
@@ -8,11 +8,13 @@
 
 {% include "Layout/Templates/header.html.twig" %}
 
+
```
Comment (bjorvack) [inline src/Backend/Core/Layout/Templates/base.html.twig]: newline isn't needed

File: src/Backend/Core/Engine/Navigation.php [hunk 2]
```diff
@@ -27,6 +27,7 @@ public function __construct(KernelInterface $kernel)
     public function parse(TwigTemplate $template): void
     {
         $template->assign('navigation', $this->navigation);
+        $template->assign('activeParent', $this->navigation[$this->getSelectedKey($this->navigation)]['label']);
```
Comment (carakas) [inline src/Backend/Core/Engine/Navigation.php]: @Katrienvh dit is niet altijd beschikbaar, op sommige pagina's geeft dit dus een error kan je dit oplossen?

Comment (StijnVrolijk) [reply src/Backend/Core/Engine/Navigation.php]: @carakas can you give an example of a page without a selected key?

Comment (carakas) [reply src/Backend/Core/Engine/Navigation.php]: the login page in the backend for instance, those pages are the reason the tests are failing

Comment (StijnVrolijk) [reply src/Backend/Core/Engine/Navigation.php]: @Katrienvh 
```
$selectedKey = $this->getSelectedKey($this->navigation);
if ($selectedKey !== null && isset($this->navigation[$selectedKey])) {
  $template->assign('activeParent', $this->navigation[$selectedKey]['label']);
}
```
And of course add an `{% if %}{% else %}{% endif %}` in your template

**Pull request merged**